### PR TITLE
Update .env.example for github callback

### PR DESCRIPTION
**Pull request by swyxio**
this seems a necessary step. altho i was still not able to get the final deployed app working at all.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -11,6 +11,7 @@ NEXTAUTH_URL=http://localhost:3000
 ## Create a GitHub OAuth app here: https://docs.github.com/en/apps/oauth-apps/building-oauth-apps/creating-an-oauth-app
 AUTH_GITHUB_ID=XXXXXXXX
 AUTH_GITHUB_SECRET=XXXXXXXX
+# you will be asked for a github callback url. it should be https://YOUR_DOMAIN.com/api/auth/callback/github
```
Comment (balazsorban44) [inline .env.example:14]: Let's point to the official documentation instead https://authjs.dev/reference/core/providers_github#callback-url

**Comment by jaredpalmer** [comment]
On Vercel? Or on another platform?

**Review by balazsorban44 — CHANGES_REQUESTED** [review]

**Comment by balazsorban44** [comment]
#28 seems related

**Comment by jaredpalmer** [comment]
@swyxio I have also reverted to an early version of `next-auth` which is confirmed to work. Have also made the `Login` in the header link to a `sign-in` page (serverless) which works fine. 

@balazsorban44 please investigate `signIn()` at edge when you wake up with latest next-auth.

**Comment by swyxio** [comment]
@jaredpalmer yes on vercel. the same 401 unauthorized issue that others have reported in #28. i just figured i'd add this to the setup instructions to save future people some time, happy to have @balazsorban44 reword it to his preference

File: .env.example (hunk 1 above)
Comment (swyxio) [reply .env.example:14]:
```suggestion
# you will be asked for a github callback url. it should be https://YOUR_DOMAIN.com/api/auth/callback/github. See docs: [authjs.dev/reference/core/providers_github#callback-url](https://authjs.dev/reference/core/providers_github#callback-url)
```

Comment (balazsorban44) [reply .env.example:14]: Could you apply this change? I don't have permission to do so. :pray: 

```suggestion
# When setting up the OAuth app, you will be asked for a GitHub callback URL. It should be https://YOUR_DOMAIN.com/api/auth/callback/github. See docs: https://authjs.dev/reference/core/providers_github#callback-url
```

**Comment by swyxio** [comment]
chat is still not working after the next auth fix https://www.loom.com/share/68bb20215d3046f8bfa6fb6906f4102f
